internal/controller/scwexternalip_controller.go

```diff
@@ -54,16 +57,27 @@ var (
 	finalizer    = "ptrk.io/finalizer"
 )
 
+const (
+	cacheKey string = "scwIpCache"
+)
+
 type NodeNameID struct {
-	Name string
-	ID   string
+	Name   string
+	ID     string
+	EIpIds []string
```

Comment:
just a nit but I'd prefer EIPIDs or ExternalIPIDs (golang way yay), but the struct can be private here if I'm not mistaken, so it could be `eipIDs` which seems best imho